jax/numpy/linalg.py

```diff
@@ -136,6 +136,32 @@ def eigvalsh(a, UPLO='L'):
   return w
 
 
+@_wraps(onp.linalg.pinv)
+def pinv(a, rcond=None):
+  # ported from https://github.com/numpy/numpy/blob/v1.17.0/numpy/linalg/linalg.py#L1890-L1979
+  a = np.conj(a)
+  if rcond is None:
+      if np.ndim(a) < 2:
```

Comment:
I don't think pinv is well defined on 1d arrays, so you should not need this logic.